core/src/main/java/org/apache/carbondata/core/datamap/status/DataMapStatusManager.java

```diff
@@ -53,6 +53,22 @@ private DataMapStatusManager() {
     return storageProvider.getDataMapStatusDetails();
   }
 
+  /**
+   * Get enabled datamap status details
+   * @return
+   * @throws IOException
+   */
+  public static DataMapStatusDetail[] getEnabledDataMapStatusDetails() throws IOException {
```

Comment:
I think this is unused function, no need to add

Reply:
It is used in `SummaryDatasetCatalog`